src/Function/Properties/Inverse.agda

```diff
@@ -42,3 +45,26 @@ isEquivalence = record
   ; sym = Symmetry.inverse
   ; trans = Composition.inverse
   }
+
+------------------------------------------------------------------------
+-- Conversion functions
+
+module _ (A : Setoid a ℓ₁) (B : Setoid b ℓ₂) where
+
+  Inverse⇒Bijection : Inverse A B → Bijection A B
+  Inverse⇒Bijection record { f = f ; f⁻¹ = f⁻¹ ; cong₁ = cong₁ ; cong₂ = cong₂ ; inverse = inverse } = record
```

Comment:
Instead of pattern matching on the record like this, it's usually neater to simply name it `I` and append a `where open Inverse I` at the end of the proof.